Server/Server.cs

```diff
@@ -61,13 +61,17 @@ private void Run()
 
             if (connected)
             {
-                do
+                for(;;)
                 {
-                    while (!System.Console.KeyAvailable)
+                    if (!System.Console.KeyAvailable)
+                    {
+                        System.Threading.Thread.Sleep(ServerConfig.Instance.Delay);

```

Comment:
Same question like on the `Client` - I'd prefer to have a single method call in case the thread sleep will be replaced with a better solution. This will not bloat the loop.

Reply:
I replied on the other comment ;)